Library/ViewModels/ChangePasswordViewModel.swift

```diff
@@ -128,8 +131,19 @@ ChangePasswordViewModelInputs, ChangePasswordViewModelOutputs {
         } else {
           return nil
         }
-    }.skipNil()
-    .skipRepeats()
+    }
+
+    self.validationErrorLabelMessage = errorLabelMessage
+      .skipNil()
+      .skipRepeats()
+
+    let entryChanged = Signal.merge(self.newPasswordProperty.signal, self.confirmNewPasswordProperty.signal)
+
+    self.accessibilityFocusValidationErrorLabel = requirementsMet
+      .takeWhen(entryChanged)
+      .filter(isFalse)
+      .ignoreValues()
+      .filter { _ in AppEnvironment.current.isVoiceOverRunning() }
```

Comment:
Would it make sense to `filter` whether VoiceOver is running before any other operation is done on the signal? Because if VoiceOver is running we're still doing calculations just to discard it later.

So instead of
```
self.accessibilityFocusValidationErrorLabel = requirementsMet
  .takeWhen(entryChanged)     
  .filter(isFalse)
  .ignoreValues()
  .filter { _ in AppEnvironment.current.isVoiceOverRunning() }
```

do
```
self.accessibilityFocusValidationErrorLabel = requirementsMet
  .takeWhen(entryChanged)
  .filter { _ in AppEnvironment.current.isVoiceOverRunning() }
  .filter(isFalse)
  .ignoreValues()
```